lambda/src/search-app-lambda-edge/search-app-lambda-edge.js

```diff
@@ -4,10 +4,54 @@
 
 import AWS from 'aws-sdk'
 import { get } from 'lodash/object'
+import { parse } from 'cookie'
 import searchURLByRegion from './searchURLByRegion'
 
 const PRODUCTION_STAGE = 'prod'
 
+const decodeBase64 = string => {
+  return Buffer.from(string, 'base64').toString('utf8')
+}
+
+const getCookieVal = rawString => {
```

Comment:
This is no longer needed. Rather than trying to decode or interpret the cookie values at all, we're just passing the raw values on and letting the authorizer do the work.